src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -637,6 +637,7 @@ const Stream = createReactClass({
   },
 
   renderGroupNodes(ids, statsPeriod) {
+    let hasGuideAnchor = ids.length > 0 && ids.length <= 2;
```

Comment:
can you add a comment about why we show a guide anchor if there are one or 2 groups, but not otherwise?

Reply:
I think we need to check if the org has <= 2 issues, it's entirely possible a project has <= 2 issues for someone who's been using Sentry for ages (if it's easy we could take it a step further and check whether all their orgs combined have <= 2 issues, but if it's hard I think it's ok to draw a line at org).